packages/app-admin-ui/client/apolloClient.js

```diff
@@ -1,21 +1,10 @@
-import { ApolloClient } from 'apollo-client';
-import { ApolloLink } from 'apollo-link';
+import { ApolloClient, ApolloLink } from '@apollo/client';
 import { createUploadLink } from 'apollo-upload-client';
-import { onError } from 'apollo-link-error';
-import { InMemoryCache, IntrospectionFragmentMatcher } from 'apollo-cache-inmemory';
+import { onError } from '@apollo/link-error';
+import { InMemoryCache } from '@apollo/client/cache';
 
 import fetch from 'cross-fetch';
 
-// This shouldn't be necessary, but it silences an annoying error from Apollo
-// Client: https://github.com/apollographql/apollo-client/issues/3397#issuecomment-421433032
-const fragmentMatcher = new IntrospectionFragmentMatcher({
```

Comment:
Need to confirm if the error this was meant to fix is gone on 3.0, but `IntrospectionFragmentMatcher` has been removed. We're supposed to use the `possibleTypes` InMemoryCache config option instead. Will need to look into that if the error is still around.

Reply:
Yeah I was confused about this too... Do you know which queries were causing these errors? I was running these changes locally, and I couldn't reproduce any errors. 

I think the below code is the same behaviour as the deleted code - but Im not sure if we need to keep this.

```javascript
new InMemoryCache({
  possibleTypes: {
    __schema: [],
  }
})
```

Reply:
I *think* had something to do with the Unsplash field? @jesstelford would know more, I think.